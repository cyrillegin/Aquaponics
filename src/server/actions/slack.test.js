import sendSlack from './slack';

jest.mock('node-fetch', () => (url, details) => {
  expect(details.body).toMatchSnapshot();
});

describe('slack action', () => {
  it('should test the function', () => {
    sendSlack({
      sensor: {
        name: 'test',
      },
      action: {
        condition: 'test',
      },
      reading: {
        value: 1,
      },
    });
  });
});
