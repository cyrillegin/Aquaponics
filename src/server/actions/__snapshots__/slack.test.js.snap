// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`slack action should test the function 1`] = `"{\\"attachments\\":[{\\"title\\":\\"test\\",\\"text\\":\\"Sensor is undefined undefined and is out of bounds, value is currently 1\\",\\"color\\":\\"#FF0000\\"}]}"`;
